js/genkit/package.json

```
@@ -156,6 +156,12 @@
       "require": "./lib/client/index.js",
       "import": "./lib/client/index.mjs",
       "default": "./lib/client/index.js"
+    },
+    "./authPolicy": {
```

We talked about this being context, right?
```suggestion
    "./context": {
```